packages/create-vite/template-react/.eslintrc.cjs

```diff
@@ -1,13 +1,15 @@
 /* eslint-env node */
 
 module.exports = {
+  root: true,
   env: { browser: true, es2020: true },
   extends: [
     'eslint:recommended',
     'plugin:react/recommended',
     'plugin:react/jsx-runtime',
     'plugin:react-hooks/recommended',
   ],
+  ignorePatterns: ['dist'],
```

Comment:
Should we also add `.eslintrc.cjs` and remove `/* eslint-env node */` to be consistent?

Reply:
I think in most case we want to only lint source code and not config files, so I wouldn't mind if we add that ignore too.

Reply:
I guess the Vite config is somewhat fine. It would be nice to keep linting from `.` and ignore the unnecessary ones though so setups like VSCode know which files to ignore. Currently, it lints every file even if we specify `src` because it doesn't know what command we run. That was also why `/* eslint-env node */` was added.